justfile

```diff
@@ -21,6 +21,10 @@ fmt:
 format:
   cargo +$(cat ./nightly-version) fmt --all --check
 
+# Generate documentation.
+docsrs *flags:
+  RUSTDOCFLAGS="--cfg docsrs -D warnings -D rustdoc::broken-intra-doc-links" cargo +$(cat ./nightly-version) doc --all-features "{{flags}}"
```

Comment:
```suggestion
  RUSTDOCFLAGS="--cfg docsrs -D warnings -D rustdoc::broken-intra-doc-links" cargo +$(cat ./nightly-version) doc --all-features {{flags}}
```

Reply:
Needs to be fixed, other than that looks good.

Reply:
done in d07550d